packages/devtools_app/lib/src/code_size/code_size_table.dart

```diff
@@ -166,8 +168,10 @@ class CodeSizeDiffTable extends StatelessWidget {
   //                  if the table is only showing negative values.
   @override
   Widget build(BuildContext context) {
+    // Ensure root node is already expanded on load.
+    rootNode.expand();
     return TreeTable<TreemapNode>(
-      dataRoots: rootNode.children,
+      dataRoots: [rootNode],
```

Comment:
instead of doing this, let's add a parameter `autoExpandRoots` to TreeTable that defaults to true. Then handle expanding all `dataRoots` in the TreeTable code

Reply:
sg, done

Reply:
sg, done

Reply:
I meant to add this parameter to `TreeTable`, not to the code size table widgets.

Reply:
This parameter will be applicable to other TreeTable's in DevTools as well. To limit the disturbance on other TreeTable widgets, set the default to `false` and add a TODO to audit other uses of TreeTable and consider defaulting to true.